test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/TestRunner.kt

```diff
@@ -169,9 +222,81 @@ private val LANG_KOTLIN_EVAL_SKIP_LIST = listOf(
     Pair("""null comparison{sql:"`null.list` = MISSING",result:missing::null}""", ERROR_EVAL_MODE_COMPILE_OPTIONS),
     Pair("""null comparison{sql:"`null.struct` = MISSING",result:missing::null}""", ERROR_EVAL_MODE_COMPILE_OPTIONS),
     Pair("""null comparison{sql:"`null.sexp` = MISSING",result:missing::null}""", ERROR_EVAL_MODE_COMPILE_OPTIONS),
+
+    // `partiql-lang-kotlin` does not implement STRICT/ERROR mode. LEGACY typing mode outputs `NULL` rather than `MISSING`
```

Comment:
New tests were added as part of updating `partiql-lang-kotlin` to use the latest `partiql-tests` commit. Some of these tests show more SQL/PartiQL spec conformance gaps